python/dgl/homophily.py

```diff
@@ -187,3 +192,99 @@ def linkx_homophily(graph, y):
             value += max(0, same_class_deg_k / deg_k - num_nodes_k / num_nodes)
 
         return value.item() / (num_classes - 1)
+
+
+def adjusted_homophily(graph, y):
+    r"""Homophily measure recommended in `Characterizing Graph Datasets for
```

Comment:
I think it's better to keep "recommended from" phrasing, because the paper actually analyzes several homophily measures (the others are already present in DGL). What is special about adjusted homophily is that, as shown in the paper, it satisfies more desirable properties than other homophily measures, and thus the paper recommends to use adjusted homophily over other options.